### PR TITLE
ETL Validation code update and fixes

### PR DESCRIPTION
**Pull request by gsorrentino18**
#### PR description:
The main changes this PR introduces in the code are the following (@fabiocos, @parbol):

1. The MeVtoMIP conversion value to be used for the new 2-discs ETL geometry validation has been fixed to the correct value of 0.015. 

2. The filling of the occupancy plots has been modified, introducing a negative weight to discriminate between the two sides of each ETL disc. 

3. The MTDGlobalReco plugins have been corrected, merging the two separate histograms used in the evaluation of the efficiencies, one for each of the two ETL discs, into a single one. This has been done since the efficiency is evaluated as:
Tracks with an associated deposit in negative (positive) side of ETL MTD/ All tracks with -3.2 < η < -1.5 (1.5 < η < 3.2),
where the numerator does not discriminate between tracks with an associated hit in the first or second ETL disc.

#### PR validation:
The new code has been tested in the release CMSSW_11_2_0_pre10

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by jfernan2** [comment]
+1

File: Validation/MtdValidation/plugins/EtlSimHitsValidation.cc [hunk 1]
```diff
@@ -214,13 +215,19 @@ void EtlSimHitsValidation::analyze(const edm::Event& iEvent, const edm::EventSet
           convertMmToCm((hit.second).x), convertMmToCm((hit.second).y), convertMmToCm((hit.second).z));
       const auto& global_point = thedet->toGlobal(local_point);
 
+      if (topo2Dis) {
```
Comment (kpedro88) [inline Validation/MtdValidation/plugins/EtlSimHitsValidation.cc:218]: could combine these conditions:
```cpp
if (topo2Dis and detId.discSide() == 1)
  weight = -weight;
```

**Comment by fabiocos** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by jpata** [comment]
+reconstruction

- reco changes are minimal and limited to the updating of the `calibrationConstant` value
- the changes are under the `phase2_etlV4` modifier, we see no reco changes as expected (there seems to be no matrix workflow for this modifier)

**Comment by fabiocos** [comment]
@jpata this PR is supposed to affect scenarios D72 and D73, which are not in the short matrix so far (they are all available in the ```upgrade``` section of ```runTheMatrix.py```)

**Comment by civanch** [comment]
+1

**Comment by jpata** [comment]
> this PR is supposed to affect scenarios D72 and D73, which are not in the short matrix so far (they are all available in the upgrade section of runTheMatrix.py)

@fabiocos 
OK, I see it now, `phase2_etlV4 -> Phase2C11_etlV4 -> 2026D72, 2026D73`, my original grep for the modifier itself did not bring it up, thanks for pointing it out. Nevertheless, it does not affect the reco signature (minimal changes protected by ETL-specific modifiers).

**Comment by fabiocos** [comment]
@jpata in case one can run here for instance wf 33434.0, but we will not see a comparison, where the effect on digitization should be apparent. A presentation about this was given at last MTD DPG by @gsorrentino18 https://indico.cern.ch/event/984445/contributions/4146432/attachments/2161556/3647170/MTD-DPG%2011_12_2020.pdf

**Comment by kpedro88** [comment]
+upgrade

**Comment by qliphy** [comment]
+1

**Pull request merged**